chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/component/ToggleType.java

```diff
@@ -0,0 +1,19 @@
+package de.codecentric.spring.boot.chaos.monkey.component;
+
+public enum ToggleType {
```

Comment:
I would like to have this not named Toggle as this enum simply represent the attacked Component Type. 
In chaos monkey terms its more of a `WatchType` (or something similar, as I don't like that name very much either, maybe you have a better idea :smile:) as you define which Components you watch.